katti/configloader.py

```diff
@@ -145,15 +217,23 @@ def get_kattis_config(config_path: str) -> KattisConfig:
     username = config.get("user", "username")
     token = None
     url = None
+    password = None
     try:
         token = config.get("user", "token")
         url = config.get("kattis", "hostname")
+        password = config.get("user", "password")
     except configparser.NoOptionError:
         pass
     if token is None or url is None:
         print("Corrupted .kattisrc file")
         print("Please navigate to https://open.kattis.com/help/submit and download a new .kattisrc")
         print("Aborting...")
         sys.exit(1)
+    if password is None:
```

Comment:
Should not terminate if password is not in config, should just refuse to do operations that require it. My suggestion would be to have the KattisConfig object not ALWAYS contain password, then fail password-relayed methods if the user_config object is missing the password key.

Reply:
i actually dont think anything needs this right now. But if you want to add features like getting leaderboard position, getting solved problems, etc, then you'll need the password. So i'll keep the login with password functionality, but if you want to remove it you can also do that.

Reply:
okay one function uses it

Reply:
Running any katti command loads the config, and that part crashes if the line `password=xxx` is not in the file. I was hoping to have it still work even without that there, and just prevent you from using password features.

Reply:
fixed